src/modules/cli/lib/execute-command.ts

```diff
@@ -10,7 +10,7 @@ type CliCommandEventMap = {
 	failure: void;
 };
 
-class CliCommandEventEmitter extends EventEmitter {
+export class CliCommandEventEmitter extends EventEmitter {
```

Comment:
```suggestion
class CliCommandEventEmitter extends EventEmitter {
```
No need to export this class, AFAICT